src/System.ServiceModel.Syndication/Microsoft.ServiceModel.Syndication.sln

```diff
@@ -0,0 +1,28 @@
+﻿
+Microsoft Visual Studio Solution File, Format Version 12.00
+# Visual Studio 15
+VisualStudioVersion = 15.0.26430.12
+MinimumVisualStudioVersion = 10.0.40219.1
+Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "Microsoft.ServiceModel.Syndication", "src\Microsoft.ServiceModel.Syndication.csproj", "{B615835F-C286-4FB5-A2BF-C3B967AB9EC6}"
+EndProject
+Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "Microsoft.ServiceModel.Syndication.Tests", "tests\Microsoft.ServiceModel.Syndication.Tests.csproj", "{A622B2C0-DD74-4218-9CF0-F9B2E52F4E91}"
```

Comment:
I'm sure you're aware but the project references and name of this solution file should be changed from `Microsoft` to `System`.

Reply:
The csproj currently does not build in corefx. The PR is mainly for porting the code from wcf to corefx. I planned to enable the build of the project later and fix the namespace later for better code review experience.